napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -358,22 +594,43 @@ def _drag_selection_box(layer, coordinates):
     layer._set_highlight()
 
 
-def _set_drag_start(layer, coordinates):
+def _set_drag_start(
+    layer: Shapes, coordinates: Tuple[np.float, ...]
```

Comment:
same as above

Reply:
Could be that it would, I type hinted here with what is specifically event.position

Reply:
resolved with [3fdedf8](https://github.com/napari/napari/pull/5555/commits/3fdedf82c7fe2b36889ad347842451f40178db40) and [3b987c1](https://github.com/napari/napari/pull/5555/commits/3b987c12fd70e0f60b57363ecbb697351411405e)